tests/_build/support/Helper/CustomRequests.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\Helper;
+
+use Codeception\Module;
+use Codeception\Module\Laravel;
+
+final class CustomRequests extends Module
+{
+    public function submitFiles(array $files): array
+    {
+        /** @var Laravel $laravelModule */
+        $laravelModule = $this->getModule('Laravel');
+        $response = $laravelModule->_request('POST', '/api/upload-files', [], $files);
+        $response = json_decode($response, true);
+
+        $last_error = json_last_error();
+        if ($last_error !== JSON_ERROR_NONE) {
+            $this->fail("Failed to parse response from uploaded-files endpoint with json error code {$last_error}");
+        }
```

Comment:
One last thing.

Since we are on PHP 7.3+, how about using `JSON_THROW_ON_ERROR` instead?

See [this ref link](https://laravel-news.com/php-7-3-json-error-handling).